basetest.pm

```diff
@@ -164,11 +161,6 @@ sub _serialize_match {
             $na->{$i} = $a->{$i};
         }
         $na->{similarity} = int($a->{similarity} * 100);
-        if ($a->{diff}) {
-            my $imgname = sprintf("%s-%d-%s-diff%d.png", $testname, $count, $name, $diffcount++);
-            $a->{diff}->write(join('/', bmwqemu::result_dir(), $imgname));
-            $na->{diff} = $imgname;
-        }
```

Comment:
actually the diff elements are gone since 99ff3f5d9b572e779b7ddbeb303cb29862925d39 - Jul 2014